ansible/roles/monit/tasks/main.yml

```diff
@@ -14,5 +14,7 @@
 
 - name: start monit
   become: yes
-  service: name="monit" state=started enabled=yes
-  tags: monit
+  service: name="monit" state=started enabled=no
```

Comment:
why the change from `enabled=yes` to `enabled=no`?

Reply:
enabled means autostart, if we start monit as soon as ubuntu boots (with no other service started) monit will keep trying and failing to start services